corehq/ex-submodules/couchforms/analytics.py

```diff
@@ -265,22 +267,69 @@ def get_form_analytics_metadata(domain, app_id, xmlns):
     }
     """
     # todo: wrap this return value in a class/stucture
-    view_results = XFormInstance.get_db().view(
-        'exports_forms/by_xmlns',
+    from corehq.apps.app_manager.models import Application
+    view_results = Application.get_db().view(
+        'exports_forms_by_app/view',
         key=[domain, app_id, xmlns],
         stale=stale_ok(),
         group=True
     ).one()
+    form_count = get_form_count_for_domain_app_xmlns(domain, app_id, xmlns)
     if view_results:
-        return view_results['value']
-    return None
+        result = view_results['value']
+        result['submissions'] = form_count
```

Comment:
How did this work before?  What about emits that matched from Application?  The `value` there doesn't have `'submissions'`